vnext/Desktop.IntegrationTests/WebSocketServer.cpp

```diff
@@ -0,0 +1,390 @@
+#include "WebSocketServer.h"
+
+#include <boost/asio/bind_executor.hpp>
+#include <boost/beast/core/buffers_to_string.hpp>
+
+using namespace boost::asio;
+
+using boost::system::error_code;
+using std::function;
+using std::placeholders::_1;
+using std::placeholders::_2;
+using std::string;
+
+namespace websocket = boost::beast::websocket;
+
+namespace Microsoft {
```

Comment:
[nit] cpp17 allows nested namespaces.

Reply:
Not sure if we're using C++17 for Desktop projects already, but I'll take a look.

Reply:
@JunielKatarn If we're accepting xlang/C++WinRT as the way to build an ABI, then I'd be introducing the cpp17 standard as a dependency of that tool set.

Reply:
Noted! I'll keep an eye out and update sources accordingly.